docs/FAQ.md

```diff
@@ -259,12 +259,16 @@ efficient to stay on the CPU instead of going back and forth.
 
 Yes, DPP still works.  It might not be as efficient as it could be, and we are working to improve it.
 
+DPP is not supported on Databricks with the plugin.
+
 ### Is Adaptive Query Execution (AQE) Supported?
 
 In the 0.2 release, AQE is supported but all exchanges will default to the CPU.  As of the 0.3 
 release, running on Spark 3.0.1 and higher any operation that is supported on GPU will now stay on 
 the GPU when AQE is enabled. 
 
+AQE is not supported on Databricks with the plugin.
```

Comment:
Similarly need a bit more detail on what "not supported" means.

Reply:
@jlowe I added the line saying AQE may fail queries on Databricks.